cmd/server-mux.go

```diff
@@ -457,6 +460,8 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 			// Execute registered handlers, update currentReqs to keep
 			// tracks of current requests currently processed by the server
 			atomic.AddInt32(&m.currentReqs, 1)
+			lmsg := fmt.Sprintf("serving for %s", r.URL.String())
+			log.logger.Info(lmsg)
```

Comment:
All you need is perhaps just this which follows apache common logging format.  You can apply this to your fork for the time being while we address the issue of logging properly. 

```diff
$ git diff
diff --git a/cmd/generic-handlers.go b/cmd/generic-handlers.go
index facef3f..962c1ec 100644
--- a/cmd/generic-handlers.go
+++ b/cmd/generic-handlers.go
@@ -20,10 +20,12 @@ import (
        "bufio"
        "net"
        "net/http"
+       "os"
        "strings"
        "time"
 
        humanize "github.com/dustin/go-humanize"
+       "github.com/gorilla/handlers"
        router "github.com/gorilla/mux"
        "github.com/rs/cors"
 )
@@ -419,3 +421,7 @@ func (h httpStatsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
        // Update http statistics
        globalHTTPStats.updateStats(r, ww, durationSecs)
 }
+
+func setLoggingHandler(h http.Handler) http.Handler {
+       return handlers.LoggingHandler(os.Stdout, h)
+}
diff --git a/cmd/routers.go b/cmd/routers.go
index 734f643..e10ae9d 100644
--- a/cmd/routers.go
+++ b/cmd/routers.go
@@ -85,6 +85,7 @@ func configureServerHandler(endpoints EndpointList) (http.Handler, error) {
 
        // List of some generic handlers which are applied for all incoming requests.
        var handlerFns = []HandlerFunc{
+               setLoggingHandler,
                // Network statistics
                setHTTPStatsHandler,
                // Limits all requests size to a maximum fixed limit
```